package.json

```diff
@@ -0,0 +1,9 @@
+{
+    "name": "express basic site",
```

Comment:
Note that this is an invalid name (you can't use spaces). It prevents from installing your project dependencies with `npm install`